app/src/core/design/components/userComponents/UtilComponents/VideoProperties.js

```diff
@@ -0,0 +1,225 @@
+import { TextField, Box, Button, InputAdornment } from "@material-ui/core";
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import { CustomAccordion } from "./Accordion";
+import { IconButton } from "@material-ui/core";
+import { Tooltip } from "@material-ui/core";
+import YouTubeIcon from "@material-ui/icons/YouTube";
+import exportImageUrl from "../../../../api/exportImageUrl";
+import CheckCircleOutlineOutlinedIcon from "@material-ui/icons/CheckCircleOutlineOutlined";
+import ReplayOutlinedIcon from "@material-ui/icons/ReplayOutlined";
+
+export function MediaAccordion({ props, setProp, src, thumbnailSrc, type, defaultThumbnail }) {
```

Comment:
since this MediaAccordion is for Video itself, we dont need to pass type and use it. its redundant